src/entity/Activity.java

```diff
@@ -0,0 +1,6 @@
+package entity;
+
+public abstract class Activity { // e ok sa o fac abstracta?
+    // private String type; // curs, serbare, excursie    nu cred ca e nevoie de acest atribut, poate o las fara niciun atribut propriu
+
```

Comment:
clasa nu contine nimic in urma comentariului tau si nu prea isi are rostul o clasa abstracta goala

Reply:
voi face sa contina ceva, aceea era ideea initiala care apoi nu mi s-a mai parut potrivita si de aceea am comentat